### ISSUE TITLE
[v1] focus in Q-field no well manage for firefox under version 52

### ISSUE THREAD

**Issue by mam-ijinus** [quasarframework/quasar#3740]
Hi all,

I don't know the minimum version of firefox requierd by Quasar 1.0 so this is peraps not a bug.
I use a web module which is under firefox45 so I can't change it.
 
### Software version
Operating System - Windows_NT(10.0.17134) - win32/x64
NodeJs - 8.11.3

**Global packages**
  NPM - 6.1.0
  yarn - 1.7.0
  @quasar/cli - 1.0.0-beta.3
  cordova - 8.0.0

**Important local packages**
  quasar - 1.0.0-beta.13 -- High performance, Material Design 2, full front end stack with Vue.js -- build SPA, SSR, PWA, Hybrid Mobile Apps and Electron apps, all simultaneously using the same codebase
  @quasar/app - 1.0.0-beta.13 -- Quasar Framework App CLI
  @quasar/extras - 1.1.1 -- Quasar Framework fonts, icons and animations
  vue - 2.6.10 -- Reactive, component-oriented view layer for modern web interfaces.
  vue-router - 3.0.2 -- Official router for Vue.js 2
  vuex - 3.1.0 -- state management for Vue.js

### What did you get as the error?
At focus on _q-field_ or _q-select_ the border does not change color to the primary color. The border stay black 1px.
in 0.17 thats work.

The issue is that to add the 'q-field--focused' class, a listener is attached to the _focusin_ event. 
The problem is that this Event only appear with firefox version 52.

## Workaround
To retrieve the behavior I create a directive: v-focus I add it to the q-input and q-select

````javascript

export default ({ Vue }) => {
  Vue.directive('focus', {
    bind: function (el, binding, node) {
      console.log('bind', node)
      let element2listen

      if (node.componentInstance.$refs.input) {
        element2listen = node.componentInstance.$refs.input
      } else if (node.componentInstance.$refs.target) {
        element2listen = node.componentInstance.$refs.target
      }

      if (element2listen) {
        element2listen.addEventListener('focus', (e) => {
          node.componentInstance.__onControlFocusin(e)
        })
        element2listen.addEventListener('blur', (e) => {
          node.componentInstance.__onControlFocusout(e)
        })
      }
    }
// unbind: function (el, binding, node) {
//    I dont know if i realy need to remove listener and how to do that with inline function...
// })
}

**Comment by mam-ijinus** [comment]
For the information, I was using [geckofx-45](https://bitbucket.org/geckofx/geckofx-45.0/wiki/Home) but they just released a new version with firefox 60 [geckofx-60.0](https://bitbucket.org/geckofx/geckofx-60.0) which is compatible with focusin event.

**Comment by rstoenescu** [comment]
Hi,

FF is considered an ever-green browser and as a result we try to support latest 4 versions of it.
